### PR TITLE
bugFix: fix a compilation error

### PR DESCRIPTION
**Pull request by sonalikatara**
##Issue Number:
None
## Description

I fixed the location of the App class in  index.js file 
## Types of changes

- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

🚨Please review the [contribution guidelines](https://github.com/gwg-women/gwg-women-techmakers/wiki/Contribution-Guidelines) to this repository.
- [ x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] Created a new branch from 'dev' and PR is targeting the project's 'dev' branch.
- [ ] The issue can be closed.

 Provide a descriptive branch name and once the pull request is approved and merged to dev, the branch can be deleted.

💔Thank you!

### CODE REVIEW

**Comment by amr08** [comment]
@sonalikatara No worries, let me rephrase.

Since we wont be using the `./build` folder in the development branch we wouldn't want to commit it here. (We'll use build in master for deploying to github pages.  Github pages requires you to reserve the master for deploying. This means, master should really only have the build files (not all the files in dev! haha but we can deal with that later)

Anyway, if you add `./build` to the .gitignore then this will prevent it from being merged by others in the future.  

Regarding the commit, the PR for fixing "app" to "App" it totally fine!  I dont think there's any regulation on the size of PRs that I know of :)

Before merging I would say make these two changes:

1. Add ./build to the .gitignore file
2. Remove build files from your Branch: 

To do this, In your terminal, type:

```
      git rm --cached build
      git status    (to make sure build is out)
      git commit -m "your message"
      git push origin fixAppFileError
```

**Comment by khusbuchandra** [comment]
Thanks @amr08 for helping out in review.
@sonalikatara , if you have already done some more changes you can always stash it by 'git stash' and  do the changes and commit as told by Andrea. After that you can git stash apply and continue on new task.

**Comment by sonalikatara** [comment]
Thank You @amr08 Now I got it. 
@khusbuchandra Thank you for the tip, I had checked out a new branch for PWA feature, didn't know about git stash.

**Comment by amr08** [comment]
Hey @khusbuchandra Is there a process for merging? ex: Maintainers only?

**Comment by khusbuchandra** [comment]
There is nothing yet defined officially. Its just based on default github account privileges. I would check and get back to you.

**Comment by khusbuchandra** [comment]
@amr08 , with write access, all the team members would be having the access. We currently don't have the process defined.
